jasmine/spec/inverted-index-test.js

```diff
@@ -1,53 +1,67 @@
-/* Setups goes here */
-const sampleBookData = [
+/* Sample Test Data */
+
+const book = [
```

Comment:
It would be cleaner if you require the books.json file here instead of hard-coding the book object, something like: `const book = require('../books.json');`